provider/testdata/data/json/list_formatter/list@1/en.json

```diff
@@ -0,0 +1,388 @@
+{
+  "patterns": {
```

Comment:
I'm not sure where you are tracking follow-ups, but please make sure there is one tracking opportunities to use same-struct fallbacking (storing Option values)

Reply:
https://github.com/unicode-org/icu4x/issues/800